src/test/java/org/cactoos/io/AppendToTest.java

```diff
@@ -53,10 +53,10 @@
      * Ensures that AppendTo is failing on a negative predicate result.
      */
     @Test
-    public void failsIfFileDoesNotExist() {
+    public void failsIfFileDoesNotExist() throws Exception {
         final File source = new File(
             new Randomized(
-            'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'
+                'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'
```

Comment:
@ccostin93 there is something strange happening here with indentation, in the previous and following lines, can you fix it so that all is correct?

Reply:
@victornoel I am using the standard intellij config.

To be honest, I've never seen the indent style before in Java. And as far as I can see in the code of the project, everything else is correctly indented and only this line seems to have had a wrong indent before, which was now fixed since I always reformat and optimize imports for all touched files.

```
new Randomized(
    'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'
).asString()
```
looks better and is easier to follow than
```
new Randomized(
'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'
).asString()
```

Example from another part of the code
<img width="606" alt="Screenshot 2020-08-07 at 22 09 50" src="https://user-images.githubusercontent.com/4803137/89684515-e8f4c380-d8fa-11ea-897a-d136cdc7bee1.png">
You can see that the code is indented the same as I indented it now.

Please let me know what you think and if I should revert the change. But imho, this is the correct formatting the code should have.

Reply:
@ccostin93 it's alright yes, I got confused by the diff, it looked like `new Randomized(` wasn't indented at all. Thx